src/idm/entity.js

```diff
@@ -0,0 +1,175 @@
+        * @param {String} entityType - type of entity
+        * @param {object} entity - An object containing the entity
+        * @fulfil {Object} entity created
+        * @returns {Promise}
+        * @example
+        * agile.idm.entity.createEntity('1','/sensor',{"name":"entity's name"}).then(function(result) {
+        *   console.log('entity created!'+result);
+        * });
+        **/
+        createEntity: (entity_id, entity_type, entity) => axios({
+          method: 'POST',
+          url: `${base}/api/v1/entity/${entity_type}/${entity_id}`,
+          headers: {"Authorization" : `bearer ${token}`},
+          data: entity
+        })
+        .then(res => (res.data))
+        .catch(errorHandler),
+    /**
+    * @summary Delete entity
+    * @name deleteEntity
+    * @public
+    * @function
+    * @memberof agile.idm.entity
+    * @param {String} entityId - id of entity
+    * @param {String} entityType - type of entity
+    * @fulfil {Undefined}
+    * @returns {Promise}
+    * @example
+    * agile.idm.entity.deleteEntity('1','/sensor').then(function() {
+    *   console.log('group removed!');
+    * });
+    **/
+    deleteEntity: (entity_id, entity_type) => axios({
+      method: 'DELETE',
+      url: `${base}/api/v1/entity/${entity_type}/${entity_id}`,
+      headers: {"Authorization" : `bearer ${token}`}
+    })
+    .then(res => (res.data))
+    .catch(errorHandler),
+    /**
+    * @summary Set Entity's attribute
+    * @name setEntityAttribute
+    * @public
+    * @function
+    * @memberof agile.idm.entity
+    * @param {String} entityId - id of entity
+    * @param {String} entityType - type of entity
+    * @param {String} attributeName- name of the attribute
+    * @param {Object|String} attribute - An object or a String containing the entity's attribute value
+    * @fulfil {Object} entity created
+    * @returns {Promise}
+    * @example
+    * agile.idm.entity.setEntityAttribute('1','/sensor',"credentials",{"dropbox":"entity's credentials for drop"}).then(function(result) {
+    *   console.log('entity created!'+result);
+    * });
+    **/
+    setEntityAttribute: (entity_id, entity_type, attribute_type, attribute_value) => axios({
+      method: 'PUT',
+      url: `${base}/api/v1/entity/${entity_type}/${entity_id}/attribute/${attribute_type}/`,
+      headers: {"Authorization" : `bearer ${token}`},
+      data: { "value" : attribute_value}
+    })
+    .then(res => (res.data))
+    .catch(errorHandler),
+    /**
+    * @summary Delete Entity's attribute
+    * @name deleteEntityAttribute
+    * @public
+    * @function
+    * @memberof agile.idm.entity
+    * @param {String} entityId - id of entity
+    * @param {String} entityType - type of entity
+    * @param {String} attributeName- name of the attribute
+    * @fulfil {Object} entity updated entity
+    * @returns {Promise}
+    * @example
+    * agile.idm.entity.deleteEntityAttribute('1','/sensor',"credentials").then(function(result) {
```

Comment:
Can set we use just `'` single quotes for strings. You can run `npm lint` to see the inconsistencies. 

You can also use eslint to fix minor issues with `--fix`.